spanner/tests/unit/test_session.py

```diff
@@ -397,6 +399,8 @@ def test_execute_sql_explicit(self):
             params,
             param_types,
             'PLAN',
+            timeout=google.api_core.gapic_v1.method.DEFAULT,
+            retry=google.api_core.gapic_v1.method.DEFAULT,
```

Comment:
The `_explicit` version of the test should pass / assert non-default values for `timeout` / `retry`.

Reply:
80e22369b5fe1d5ddb6a9f00a0663e5acd7ac6f2